### PR TITLE
SDK 479 changes - request object

### PR DESCRIPTION
**Pull request by fundthmcalculus**

### CODE REVIEW

File: Sources/Trinsic/WalletService.swift [hunk 1]
```diff
@@ -22,10 +22,11 @@ public class WalletService : ServiceBase
         client = Services_Universalwallet_V1_UniversalWalletClient(channel: createChannel())
     }
     
-    public func search(query: String = "SELECT * from c") throws -> Services_Universalwallet_V1_SearchResponse {
-        var request = Services_Universalwallet_V1_SearchRequest();
-        request.query = query;
-        
+    public func search(request: Services_Universalwallet_V1_SearchRequest()) throws -> Services_Universalwallet_V1_SearchResponse {
```
Comment (tmarkovski) [inline Sources/Trinsic/WalletService.swift:25]: This fails compilation, it's invalid swift syntax

**Review by tmarkovski — APPROVED** [review]

**Comment by tmarkovski** [comment]
@fundthmcalculus can this be merged?

**Comment by fundthmcalculus** [comment]
Yes

**Review by tmarkovski — APPROVED** [review]

**Comment by tmarkovski** [comment]
There are conflicts to be resolved

**Pull request merged**